CONTRIBUTING.rst

```diff
@@ -104,31 +106,30 @@ Set up to run tests
 *******************
 
 To run the tests for the first time, you will need to do some setup (after
-this, you can just work on your given virtualenv each time)::
+this, any ``uv`` commands will automatically work within the right virtual
+environment)::
 
-    python -m venv .venv
-    . .venv/bin/activate
+    uv sync --all-extras
 
-    pip install pytest pytest-xdist
-    pip install -c requirements.txt -e .[aafiguresupport,mathsupport,plantumlsupport,rawhtmlsupport,sphinx,svgsupport,tests]
+If you don't have ``uv``, please see `the installation docs <https://docs.astral.sh/uv/getting-started/installation/>`_
 
 Run tests
 *********
 
 To run all tests, run::
 
-  pytest
+    uv run pytest
```

Comment:
Instead of having `pytest` in your global path (inside the virtualenv), prefixing commands with `uv run` ensures they're running within the context of your project's environment without any ambiguity.